components/classes/experienceevent.schema.json

```diff
@@ -27,7 +27,6 @@
       }
     }
   },
-  "required": ["@id", "xdm:timestamp"],
```

Comment:
Not sure we can do this. We may be able to move this required attribute to the time-series behavior which is where these fields come from.